install-cloud.sh

```diff
@@ -11,6 +11,14 @@
 
 set -e
 
+if [ -f /etc/centos-release ]
```

Comment:
The generic version of this would be doing something similar on each distro.  An install command for an existing package is a no-op, so there's no need to check.  But the install command (and package name) varies per distribution.